index.js

```diff
@@ -195,6 +196,10 @@ Paper.prototype.cdnify = function (path) {
         path = path.substr(8, path.length);
     }
 
+    if (sessionId) {
+        return [cdnUrl, 'stencil', versionId, configId, sessionId, path].join('/');
```

Comment:
that won't work because `cdnUrl` is a url. And the output will be:
```
      "http:/cdn.example.com/foo/stencil/123/234/345/image.jpg"
```

notice the `http:/` part